### PR TITLE
[MTT-8705] Upgrade Client Driven Sample to Unity 6 

### PR DESCRIPTION
**Pull request by FabianStoll**
### Description
This PR is for upgrading the Client Driven Sample to Unity 6.
It also replaces ParrelSync with MPPM and fixes any warnings that pop up during startup.

### Issue Number(s)
[MTT-8705](https://jira.unity3d.com/browse/MTT-8705)

### Contribution checklist
 - [ ] Tests have been added for the project and/or any internal package
 - [x] Release notes have been added to the [project changelog](../CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] JIRA ticket ID is in the PR title or at least one commit message
 - [x] Include the ticket ID number within the body message of the PR to create a hyperlink

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize?pullRequest=259) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize?pullRequest=259) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: hammerlerobi<br/>:x: FabianStoll<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.multiplayer.samples.bitesize?pullRequest=259) it.</sub>

File: Basic/ClientDriven/Assets/Materials/Black.mat [hunk 1]
```diff
@@ -19,7 +19,8 @@ Material:
   m_CustomRenderQueue: 2000
   stringTagMap:
     RenderType: Opaque
-  disabledShaderPasses: []
+  disabledShaderPasses:
+  - MOTIONVECTORS
```
Comment (Elfi0Kuhndorf) [inline Basic/ClientDriven/Assets/Materials/Black.mat:23]: Were the materials all automatically updated or did you manually change it?

File: Basic/ClientDriven/ProjectSettings/GraphicsSettings.asset [hunk 2]
```diff
@@ -3,7 +3,7 @@
 --- !u!30 &1
 GraphicsSettings:
   m_ObjectHideFlags: 0
-  serializedVersion: 14
+  serializedVersion: 16
```
Comment (Elfi0Kuhndorf) [inline Basic/ClientDriven/ProjectSettings/GraphicsSettings.asset:6]: Did the graphics settings changes come automatically through the package upgrade?

**Review by Elfi0Kuhndorf — APPROVED** [review]
Looks good to me! I also tested in the editor with multiple MPPM windows and it worked all as expected.

**Pull request merged**